scss/modules/_nav.scss

```diff
@@ -297,92 +241,134 @@
       background-position: 90% 1rem;
     }
   }
-
-  @include media($lg) {
-    display: none;
-  }
 }
 
-.site-nav__button--right {
-  @include u-background-image('i-book--secondary');
-  float: right;
-  height: 3.6rem;
-
-  &:hover {
-    @include u-background-image('i-book--neutral');    
-    background-color: lighten($secondary, 10%);
-  }  
+// BREAKPOINT: LARGE
+// Switches to traditional dropdown menu
+// Hide the submenu toggles
 
-  @include media($lg) {
-    height: 6rem;
+@include media($lg) {
+  .site-nav {
+    padding-left: 2rem;
   }
-}
 
-// Toggle controls for the mobile menu
-// 
-// We use the same logic to show the entire menu as well as the sub-menus (site-nav__dropdown)
-// When on mobile, we hide the site-nav__link in the site-nav__item--with-dropdown elements so that tapping on the toggle opens the dropdown, instead of taking you to the page
-//
-// Elements:
-// .nav-toggle__input                       - The checkbox that controls whether adjacent lists are shown or hidden
-// .nav-toggle__label                       - The label attached to the checkbox (not applied to the main menu button)
-// .site-nav__item--with-dropdown           - The outer li that contains the dropdown list and toggle
+  .site-nav__list {
+    @include clearfix();
+    @include u-bg--neutral();
+    @include transform(translateX(0));
+    background: none;
+    border: none;
+    min-height: 0;
+    display: block;
+    float: left;
+    padding: 0;
+    position: relative;
+    top: 0;
 
-.site-nav__item--with-dropdown {
-  & > a.site-nav__link {
-    display: none;
+    .utility-nav__item {
+      display: none;
+    }
   }
 
-  @include media($lg) {
-    & > a.site-nav__link {
-      display: block;
+  .site-nav__item {
+    float: left;
+    border-bottom: none;
+
+    &.is-open,
+    &:hover {
+      .site-nav__link {
+        border-bottom-color: $primary-contrast;
+        background-color: $primary;
+        color: $ivory;
+      }
     }
 
-    & > .nav-toggle__input,
-    & > .nav-toggle__label {
-      display: none;
+    &.is-disabled {
+      .site-nav__link {
+        color: #737373;
```

Comment:
Color literals like `#737373` should only be used in variable declarations; they should be referred to via variable everywhere else.